js/client.js

```diff
@@ -1158,6 +1161,15 @@ function toId() {
 				break;
 			}
 		},
+		saveIgnore: function () {
+			var buf = {};
+			if (!this.ignore) return; // ??
+			for (var k in this.ignore) {
+				// we set this to false so they're notified once per session of it
+				buf[k] = {notified: false};
```

Comment:
This seems like it would be notify more than once per session, but rather every time it was saved?

Reply:
No, because buf isn't app.ignore. 
it just ensures that next time they start a new session, they'll be notified again.

Reply:
Please don't resolve conversations after you answer the question, that makes it so I can't see the answer. I'll resolve it once I see the answer. :[

Reply:
It seems like it's better to just set it to `1` and then handle it in the loader?

Reply:
Okay. Done.